pkg/cmd/setup.go

```diff
@@ -63,106 +64,118 @@ func setup(cmd *cobra.Command, args []string) {
 		utils.LogDebugError(err.Unwrap())
 	}
 
-	ignoreRepoConfig :=
-		// ignore when repo config is blank
-		(repoConfig.Setup.Project == "" && repoConfig.Setup.Config == "") ||
-			// ignore when project and config are already specified
-			(localConfig.EnclaveProject.Source == models.FlagSource.String() && localConfig.EnclaveConfig.Source == models.FlagSource.String())
-
-	// default to true so repo config is used on --no-interactive
-	useRepoConfig := true
-	if !ignoreRepoConfig && canPromptUser {
-		useRepoConfig = utils.ConfirmationPrompt("Use settings from repo config file (doppler.yaml)?", true)
-	}
-
-	currentProject := localConfig.EnclaveProject.Value
-	selectedProject := ""
-
-	switch localConfig.EnclaveProject.Source {
-	case models.FlagSource.String():
-		selectedProject = localConfig.EnclaveProject.Value
-	case models.EnvironmentSource.String():
-		utils.Log(valueFromEnvironmentNotice("DOPPLER_PROJECT"))
-		selectedProject = localConfig.EnclaveProject.Value
-	default:
-		if useRepoConfig && repoConfig.Setup.Project != "" {
-			utils.Print("Auto-selecting project from repo config file")
-			selectedProject = repoConfig.Setup.Project
-			break
+	for _, repo := range repoConfig.Setup {
+		if len(repoConfig.Setup) > 1 && repo.Path == "" {
+			utils.HandleError(errors.New("a path must be specified for all repos when more than one exists in the repo config file (doppler.yaml)"))
 		}
-
-		projects, httpErr := http.GetProjects(localConfig.APIHost.Value, utils.GetBool(localConfig.VerifyTLS.Value, true), localConfig.Token.Value, 1, 100)
-		if !httpErr.IsNil() {
-			utils.HandleError(httpErr.Unwrap(), httpErr.Message)
-		}
-		if len(projects) == 0 {
-			utils.HandleError(errors.New("you do not have access to any projects"))
+		expandedPath, _ := filepath.Abs(repo.Path)
```

Comment:
What does `filepath.Abs()` return when `repo.Path` is an empty string?

Reply:
From the docs:
>  If the path is empty, Base returns "."

Reply:
Oh wait, that's for a different function. Pretty sure it works the same. Let me double-check though.

Reply:
Confirmed – it uses the current working directory.

Reply:
https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/path/filepath/path.go;l=304-308
https://cs.opensource.google/go/go/+/refs/tags/go1.20.2:src/path/filepath/path_windows.go;l=210-215